### ISSUE TITLE
DXF mangled into non-manifold geometry

### ISSUE THREAD

**Issue by stuartpb** [openscad/openscad#2276]
Check out https://github.com/stuartpb/tubman20-stamp/tree/63ef15e05a4610708245ae060e63cc1147ad6275, which extrudes [this DXF](https://github.com/stuartpb/tubman20-stamp/blob/63ef15e05a4610708245ae060e63cc1147ad6275/jackson-silhouette.dxf) in [tubman20-stamp.scad](https://github.com/stuartpb/tubman20-stamp/blob/63ef15e05a4610708245ae060e63cc1147ad6275/tubman20-stamp.scad) - though the DXF appears to be fine (based on a cursory examination in FreeCAD), the resulting STL has a bunch of gross overlapping polygons that make it unprintable.

Behavior observed in version 2018.01.06 (git 3473eb1).

**Comment by t-paul** [comment]
This DXF extrudes fine for me with both release and nightly build on Linux using
```
difference() {
    linear_extrude(1) import("/tmp/jackson-silhouette.dxf");
    translate([6, 3, 0]) cylinder(r = 2, h = 10, center = true);
}
```

**Comment by thehans** [comment]
Looking at one of the DXF by itself, seems that it has a bunch of self intersecting lines which I would assume are what throws off OpenSCAD.

![screenshot from 2018-01-31 09-48-20](https://user-images.githubusercontent.com/252233/35632327-fdf8eb58-066b-11e8-8db0-9decce60763f.png)

**Comment by stuartpb** [comment]
Yeah, I meant `tubman-portrait.dxf`, not `jackson-silhouette.dxf`.

**Comment by stuartpb** [comment]
What's self-intersecting about the lines in tubman-portrait.dxf? They look like normal, proper holes to me: I don't see any lines overlapping.

**Comment by kintel** [comment]
Status as of 2024-03-24: Works when using the Manifold backend
